Packs/PrismaCloudComputeReporting/Playbooks/Prisma_Cloud_Compute_Vulnerability_and_Compliance_Reporting.yml

```diff
@@ -0,0 +1,231 @@
+      description: start
+      name: ""
+      version: -1
+    taskid: 4c967bc6-4854-403e-8ec5-8db149022e2d
+    timertriggers: []
+    type: start
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 50
+        }
+      }
+  "1":
+    id: "1"
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    nexttasks:
+      '#none#':
+      - "2"
+    note: false
+    quietmode: 0
+    scriptarguments:
+      compact:
+        simple: "false"
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: PaloAltoNetworks_PrismaCloudCompute
+      description: Get images scan report. The report includes vulnerabilities, compliance issues, binaries, etc.
+      id: 8dde8a67-ebf4-49a0-8448-475f2919fbbf
+      iscommand: true
+      name: Pull Image Details
+      script: PaloAltoNetworks_PrismaCloudCompute|||prisma-cloud-compute-images-scan-list
+      type: regular
+      version: -1
+    taskid: 8dde8a67-ebf4-49a0-8448-475f2919fbbf
+    timertriggers: []
+    type: regular
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 190
+        }
+      }
+  "2":
+    id: "2"
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    nexttasks:
+      '#none#':
+      - "3"
+    note: false
+    quietmode: 0
+    scriptarguments:
+      cvedescription:
+        simple: ${PrismaCloudCompute.ReportsImagesScan.vulnerabilities.description}
+      cvemodified:
+        simple: ${PrismaCloudCompute.ReportsImagesScan.vulnerabilities.fixDate}
+      cvss:
+        simple: ${PrismaCloudCompute.ReportsImagesScan.vulnerabilities.cvss}
+      type:
+        simple: CVE
+      value:
+        simple: ${PrismaCloudCompute.ReportsImagesScan.vulnerabilities.cve}
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: Builtin
+      description: commands.local.cmd.new.indicator
+      id: f4840f70-ab61-4f0e-8150-734ba46e2ac2
+      iscommand: true
+      name: Create CVE Indicators
+      script: Builtin|||createNewIndicator
+      type: regular
+      version: -1
+    taskid: f4840f70-ab61-4f0e-8150-734ba46e2ac2
+    timertriggers: []
+    type: regular
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 370
+        }
+      }
+  "3":
+    id: "3"
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    nexttasks:
+      '#none#':
+      - "4"
+    note: false
+    quietmode: 0
+    scriptarguments:
+      cve:
+        simple: ${CreatedIndicator.Value}
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      description: Create a markdown table of impacted images and hosts based on Prisma Cloud Compute CVE data.
+      id: 49ca785f-8c72-4efd-8e6a-f774cade1445
+      iscommand: false
+      name: Create Impact Table on CVE based on Prisma Cloud Compute Data
+      script: a0e80035-f917-4ad5-8236-54904ad5d6e6
```

Comment:
@nericksen  Please add the correct script values under the "name" and "script" keys.